style.css

```diff
@@ -413,7 +409,96 @@ body {
   font-weight: normal;
 }
 
-.btn-link {
-  border: 1px solid red;
-  float: left;
+/* contact form */
+
+.form-container {
+  background-color: #6070ff;
+  width: 100%;
+  height: 750px;
+  border: none;
+  left: 0;
+  top: 1rem;
+  border-top-left-radius: 60px;
+}
+
+.form-header {
+  font-size: 40px;
+  font-weight: 800;
+  line-height: 52px;
+  color: #fff;
+  text-align: center;
+  padding-top: 70px;
+}
+
+.form-content {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  text-align: center;
+  color: #fff;
+  margin-left: 30px;
+  margin-top: 2rem;
+  max-width: 400px;
+}
+
+li {
+  list-style-type: none;
+}
+
+input {
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  height: 48px;
+  width: 350px;
+  margin-top: 1rem;
+  display: flex;
+  outline: none;
+  gap: 1.5rem;
+  justify-content: center;
+  margin-left: 30px;
+  border-radius: 12px;
+}
+
+input::placeholder {
+  padding: 12px;
+  color: #172b4d;
+}
+
+textarea {
+  width: 350px;
+  height: 180px;
+  margin-top: 2rem;
+  margin-left: 30px;
+  border-radius: 8px;
+  border: none;
+}
+
+textarea::placeholder {
+  padding: 15px;
+  color: #b3bac5;
+}
+
+button {
+  display: block;
+  margin-top: 1rem;
+  width: 14rem;
+  height: 3.2rem;
+  color: #6070ff;
+  background-color: #fff;
+  border-radius: 15px;
+  font-size: 18px;
+  font-weight: 500;
+  border: 1px solid #7f8cff;
+  margin-left: 50px;
+}
+
+button:hover {
+  cursor: pointer;
+  color: #344563;
+  border: 1px solid #000;
+  background-color: #ddd;
 }
```

Comment:
- [ ] The Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template. Please use the some CSS properties from the Figma design look ⬇️ 

![buttongood](https://user-images.githubusercontent.com/79058364/165333032-7d562e97-53c3-4614-a452-1f680d6fcf41.png)